packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -791,20 +867,23 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   ///
   /// The `offset` is in global coordinates.
   ///
+  /// Provide the `textGranularity` if the selection should not move the default
+  /// character by character.
```

Comment:
Nit: Maybe specify that the default is `[TextGranularity.character]` instead of saying "character by character".